ajax/libs/valine/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "valine",
+  "filename": "Valine.min.js",
+  "description": "A simple comment system based on Leancloud.",
+  "version": "1.1.8",
+  "keywords": [
+    "Valine",
+    "comment",
+    "system",
+    "Leancloud",
+    "Minimalist",
+    "simple",
+    "easy-to-use",
+    "fast-and-safe",
+    "comment-system"
+  ],
+  "author": "xCss <xioveliu@gmail.com> (https://github.com/xCss)",
+  "homepage": "https://valine.js.org",
+  "license": "GPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/xcss/Valine.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/xcss/Valine.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/!(*escape*|*detect*)"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "Valine*.+(js|map|css)"
+        ]
```

Comment:
Let's remove L33-L38 because we can't use the file in web front-end. (According to https://github.com/xCss/Valine/issues/79#issuecomment-392318191)
Also, please remove `ajax/libs/valine/1.1.8/Valine.js`

Many thanks.